### ISSUE TITLE
Prevent render while page fully loaded

### ISSUE THREAD

**Issue by smokerigni** [mozilla/pdf.js#10881]
**Attach (recommended) or Link to PDF file here:**
[Large PDF](http://ftp.geogratis.gc.ca/pub/nrcan_rncan/publications/ess_sst/222/222861/mr_93_e.pdf)

**Configuration:**
- Web browser and its version: Google Chrome 74.0.3729.169
- Operating system and its version: Win 10 Pro 1809
- PDF.js version: maybe v2.0.943 (used [Online demo](http://mozilla.github.io/pdf.js/web/viewer.html))
- Is a browser extension: No

**Steps to reproduce the problem:**
1. Download a big pdf file like this: [Large PDF](http://ftp.geogratis.gc.ca/pub/nrcan_rncan/publications/ess_sst/222/222861/mr_93_e.pdf)
2. Open the file

**What is the expected behavior?**
Is there any way to prevent the unnecessary painting while the document is fully loaded? Or just a flag to process in background? Just for comparsion, Chrome's native one awaits 10 seconds, and then appers a fully loaded document, pdf.js (used locally) takes more long time (avg. ~26 seconds) 
![image](https://user-images.githubusercontent.com/6452925/58868656-0e65bc00-86bd-11e9-819f-43441ac77636.png)

**What went wrong?**
The loading progress takes long time, items appear one after another.
![image](https://user-images.githubusercontent.com/6452925/58868121-f3467c80-86bb-11e9-8f94-e7f5ded5da44.png)

**Link to a viewer:**
[Your online demo](http://mozilla.github.io/pdf.js/web/viewer.html)

Any comments and answers will be appreciated. 🎉

**Comment by timvandermeij** [comment]
Not really. This PDF file appear to contain a huge amount of small image fragments that need to be rendered. This is a performance problem and we could look into if we can improve the rendering time here.

**Comment by smokerigni** [comment]
Thank you for your answer, awaiting good news 👍

**Comment by vdnhi** [comment]
@timvandermeij is there any update for this issue?

**Comment by Snuffleupagus** [comment]
> **Attach (recommended) or Link to PDF file here:**
> [Large PDF](http://ftp.geogratis.gc.ca/pub/nrcan_rncan/publications/ess_sst/222/222861/mr_93_e.pdf)

Unfortunately that link no longer works, and the file is also not available from the Internet Archive; closing for now.
